src/main/java/org/spongepowered/api/potion/PotionEffectType.java

```diff
@@ -35,10 +36,18 @@
 public interface PotionEffectType extends CatalogType, Translatable {
 
     /**
-     * Gets whether this potion effect is applied
-     * instantly or over time.
+     * Gets whether this potion effect is applied instantly or over time.
      *
      * @return If applied instantly.
      */
     boolean isInstant();
+
+    /**
+     * Gets the {@link Translation} for this potion effect type as a potion
+     * name.
+     *
+     * @return The translation representing this effect as potion
+     */
+    Translation getPotionTranslation();
```

Comment:
I may have named this badly
potion.empty=No Effects
potion.moveSpeed=Speed
potion.moveSlowdown=Slowness
potion.digSpeed=Haste
potion.digSlowDown=Mining Fatigue
potion.damageBoost=Strength
potion.heal=Instant Health
potion.harm=Instant Damage
potion.jump=Jump Boost
potion.confusion=Nausea
potion.regeneration=Regeneration
potion.resistance=Resistance
potion.fireResistance=Fire Resistance
potion.waterBreathing=Water Breathing
potion.invisibility=Invisibility
potion.blindness=Blindness
potion.nightVision=Night Vision
potion.hunger=Hunger
potion.weakness=Weakness
potion.poison=Poison
potion.wither=Wither
potion.healthBoost=Health Boost
potion.absorption=Absorption
potion.saturation=Saturation

potion.moveSpeed.postfix=Potion of Swiftness
potion.moveSlowdown.postfix=Potion of Slowness
potion.digSpeed.postfix=Potion of Haste
potion.digSlowDown.postfix=Potion of Dullness
potion.damageBoost.postfix=Potion of Strength
potion.weakness.postfix=Potion of Weakness
potion.heal.postfix=Potion of Healing
potion.harm.postfix=Potion of Harming
potion.jump.postfix=Potion of Leaping
potion.confusion.postfix=Potion of Nausea
potion.regeneration.postfix=Potion of Regeneration
potion.resistance.postfix=Potion of Resistance
potion.fireResistance.postfix=Potion of Fire Resistance
potion.waterBreathing.postfix=Potion of Water Breathing
potion.invisibility.postfix=Potion of Invisibility
potion.blindness.postfix=Potion of Blindness
potion.nightVision.postfix=Potion of Night Vision
potion.hunger.postfix=Potion of Hunger
potion.poison.postfix=Potion of Poison
potion.wither.postfix=Potion of Decay
potion.healthBoost.postfix=Potion of Health Boost
potion.absorption.postfix=Potion of Absorption
potion.saturation.postfix=Potion of Saturation